garage/tf/core/networks.py

```diff
@@ -64,3 +65,37 @@ def mlp(input_var,
             bias_initializer=output_b_init,
             name="output")
     return l_out
+
+
+def parameter(input_var,
```

Comment:
how could a parameter have an `input_var`?

Reply:
This is used for broadcast the parameter shape.